.github/workflows/build.yml

```diff
@@ -2,7 +2,7 @@ name: Build
 
 on:
   pull_request:
-    branches: [main, develop]
+    branches: [main]
```

Comment:
Should we have develop here? Since PRs are opened against develop.

Reply:
The new workflow does this as one of its steps, so I took it out of here since it would be doing the same thing twice.